### DISCUSSION TITLE
(pdf) previewer does not show payment details (bank contact) on old invoices

### DISCUSSION THREAD

**Discussion by boonkerz** [AndresMorelos/InvoncifyReleases#45] [Q&A] [answered]
minimal template -> footer
{ payment && payment.details &&  payment.details.split('\n').map((item, i) => <p key={i}>{item}</p>) }

payment object is empty

**Comment by AndresMorelos** [comment]
hello @boonkerz, could you please share screenshots about it? 

BTW: the payment settings can be updated in the settings tab >> invoice >> Payment Details or in the invoice tab if you mark the item as required.

## Settings >> Invoice >> Payment Details
<img width="1306" alt="image" src="https://user-images.githubusercontent.com/6563162/160437166-29d05375-65a8-46aa-ba84-5d762f38f5dd.png">

## New Invoice >> Form Settings >> Payment Checked
<img width="1311" alt="image" src="https://user-images.githubusercontent.com/6563162/160437385-34046c06-b55b-406a-a496-60f5c782165a.png">

---

Let me know if you need more assistance

**Comment by boonkerz** [comment]
Ok i haven't checked the payment before.

the checkbox is does only effect new invoices earlier does will not effect.
can we change this? or should i delete older invoice and re create?

**Comment by AndresMorelos** [comment, marked as answer]
@boonkerz the setting will affect the new invoices with the current data, if you need to add the payment details to old invoices click on edit and then at form settings activate the payment, and set up the payment details. 


BTW: Moving issue to conversation.

**Reply by boonkerz** [reply]
thx found
